src/renderer/runner.ts

```diff
@@ -388,22 +388,30 @@ export class Runner {
     const options = [dir, '--inspect'].concat(executionFlags);
 
     return new Promise(async (resolve, _reject) => {
-      this.child = await fiddleRunner.spawn(
-        isValidBuild && localPath ? Installer.getExecPath(localPath) : version,
-        dir,
-        { args: options, cwd: dir, env },
-      );
+      try {
+        this.child = await fiddleRunner.spawn(
+          isValidBuild && localPath
+            ? Installer.getExecPath(localPath)
+            : version,
+          dir,
+          { args: options, cwd: dir, env },
+        );
+      } catch (e) {
+        pushOutput(`Failed to spawn Fiddle: ${e.message}`);
+        resolve(RunResult.FAILURE);
```

Comment:
```suggestion
        return resolve(RunResult.FAILURE);
```

Reply:
yes, it looks like you saw this right before i force pushed it. It's been fixed and other code changed since then.